workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -97,6 +105,16 @@ export const WorkflowInstancePageContent: React.FC<{
       delete instanceVariables.result;
     }
   }
+  const workflowId = assessedInstance.instance.processId;
+  const instanceId = assessedInstance.instance.id;
+  const { value } = useAsync(async (): Promise<InputSchemaResponseDTO> => {
```

Comment:
we always need handling of error and loading state

Reply:
In case of error add a ResponseErrorPanel
In case of loading - add a progress bar